pkg/kvstore/etcd.go

```diff
@@ -81,6 +81,9 @@ func newEtcdClient(config *client.Config, cfgPath string) (KVClient, error) {
 		}
 	}
 	if config != nil {
+		if config.DialTimeout == 0 {
+			config.DialTimeout = 10 * time.Second
```

Comment:
I think 1 minute would be better. Why not retrying forever instead of bailing out after 10 seconds?